src/main/scala/org/broadinstitute/dsde/rawls/model/UserAuth.scala

```diff
@@ -13,7 +13,7 @@ case class RawlsUserSubjectId(value: String) extends UserAuthType
 case class RawlsGroupName(value: String) extends UserAuthType
 case class RawlsGroupEmail(value: String) extends UserAuthType
 case class RawlsBillingProjectName(value: String) extends UserAuthType
-case class RawlsGroupMemberList(userEmails: Seq[String], subGroupEmails: Seq[String])
+case class RawlsGroupMemberList(userEmails: Option[Seq[String]] = None, subGroupEmails: Option[Seq[String]] = None, userSubjectIds: Option[Seq[String]] = None, subGroupNames: Option[Seq[String]] = None)
```

Comment:
More constructive commentary: I don't feel like this is as user-friendly as it could be.

My suggestion: `RawlsGroupMemberList( users: Seq[String], groups: Seq[String] )`, and determine in code if it's a subject ID (all numerals), email (contains an `@`), or name (otherwise) would be less clumsy. This assumes that `@` is not valid for group names, which I [think](https://support.google.com/a/answer/33386) is true.